### PR TITLE
MAINT: Analytics using plausible.io with scientific-python

### LINKED ISSUE #338: Add Google Analytics?
One way of measuring impact and which functions are used frequently would be via website traffic. I'd be interested in setting up Google Analytics in order to collect that data. Based on [this](https://www.similarweb.com/website/scipy.org#overview) estimate of the number of hits scipy.org gets per month (~3 million), we wouldn't exceed the [traffic limit](https://support.google.com/analytics/answer/1070983?hl=en) for Google Analytics free version (10 million hit/month). [Setup](https://support.google.com/analytics/answer/1008015?hl=en) just requires adding the same snippet of code to the `<head>` of every page we want to track.

1. Shall I give this a shot?
2. Would the right place to add the code to this repo be an [`extrahead` block](https://www.sphinx-doc.org/en/master/templating.html) in [`layout.html`](https://github.com/scipy/scipy.org/blob/master/www/_templates/layout.html)?
2. Should this be added to scipy-sphinx-theme, too? Would adding the code directly to the `<head>` section of its [`layout.html`](https://github.com/scipy/scipy-sphinx-theme/blob/master/_theme/scipy/layout.html) make the code appear on every page?

### PR DESCRIPTION
**Pull request by tupui**
Closes #338

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -20,6 +20,9 @@ params:
   font:
     name: "Lato"
     sizes: [400,900]
+  plausible:
+    dataDomain: scipy.org
+    javaScript: "https://analytics.scientific-python.org/js/plausible.js"
```
Comment (stefanv) [inline config.yaml:25]: You don't need to specify javaScript here, since that is the default.  But it won't hurt either.

**Comment by stefanv** [comment]
Note that you'd also need to pin the latest version of the theme for it to work.

File: config.yaml (hunk 1 above)
Comment (tupui) [inline config.yaml:25]:
```suggestion
```

Comment (tupui) [reply config.yaml:25]: Let's remove it then since either way it's linked to the theme.

**Review by tupui — COMMENTED** [review]
> Note that you'd also need to pin the latest version of the theme for it to work.

I updated the theme, did I miss something else?

**Comment by tupui** [comment]
How does it work to see the metrics?? Do you have a dashboard specific to SciPy then? Is this public or restricted to some people/org/teams?

**Review by stefanv — APPROVED** [review]

**Comment by stefanv** [comment]
> How does it work to see the metrics?? Do you have a dashboard specific to SciPy then? Is this public or restricted to some people/org/teams?

I will send you credentials for the SciPy dashboard. Credentials are limited to a single website, and AFAIK you can have as many users as you like.

**Comment by stefanv** [comment]
Account invitation sent to you and Ralf. You will be able to add others.

**Comment by tupui** [comment]
Thank you Stéfan 🙏 @rgommers is this ok for you?

**Comment by tupui** [comment]
The dashboard looks great! Lot of options and we could even make some data public if we wanted to.

**Comment by tupui** [comment]
I added some basic note in the readme to say that there is a dashboard.

**Review by rgommers — APPROVED** [review]
LGTM, thanks @tupui and @stefanv

**Comment by rgommers** [comment]
@stefanv could you please resend the invite to my personal email?

**Comment by tupui** [comment]
For me it was in my spam.

**Comment by mdhaber** [comment]
I'd appreciate an invitation, too! Thanks for setting this up, all.

**Comment by tupui** [comment]
Just sent it to you @mdhaber on your personal email.

**Pull request merged**